portality/forms/application_forms.py

```diff
@@ -486,16 +492,22 @@ class FieldDefinitions:
             },
             "admin": {
                 "widgets": [
+                    "trim_whitespace",  # ~~^-> TrimWhitespace:FormWidget~~
+                    {"autocomplete": {"type": "journal", "field": "bibjson.publisher.name.exact"}},
```

Comment:
Should we add `# ~~-> Autocomplete:FormWidget~~` here? (or `# ~~^-> Autocomplete:FormWidget~~`??)